k8s/airflow-operator/manifest/manifests.yaml.template

```diff
@@ -21,7 +32,30 @@ spec:
         pending: []
     spec:
       serviceAccountName: $SERVICE_ACCOUNT
+      initContainers:
+      - name: extract-crds
+        image: $IMAGE_AIRFLOWOPERATOR
+        imagePullPolicy: Always
+        command:
+        - sh
+        - -c
+        - "ls; pwd; ls /; cp crds/* /extracted/"
+        volumeMounts:
+        - name: crds
+          mountPath: /extracted
+      - name: inject-crds
+        image: gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:0.7.0
```

Comment:
Please declare a new property with x-google-marketplace type of DEPLOYER_IMAGE instead. The real deployer image will be injected there. So you can actually copy additional files to the deployer image and use it there.

```
properties:
  # As always, any name here. Should just match what you use in the template.
  DEPLOYER_IMAGE:
    type: string
    x-google-marketplace:
      type: DEPLOYER_IMAGE
```

Reply:
Which option did you suggest we do ?

Option A:
DEPLOYER_IMAGE = gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:0.7.0

Option B:
Build airflow_deployer by using  "FROM clone gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:0.7.0"
DEPLOYER_IMAGE=airflow_deployer:0.7 ?

Reply:
Literally just copy and paste the `properties` snippet above into `schema.yaml`. And the manifest should become:

```
...
- name: inject-crds
  image: $DEPLOYER_IMAGE
```